mynearwallet/my-near-wallet
# chore: convert and add storybook to components

## Description
Refactor a lot of components to be in TypeScript with accompanying stories

## Work Done
* Converted a lot of components to typescript
* Added storybook to components can be visualized
* Converted to new way of handling translations

## How to test
* Run `yarn storybook` (note I need to add redux to storybook for some of the stories to work)
* Run the app and click around (nothing should have changed)

## Screenshots
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/46312751/233005015-9d53201f-0105-4e89-aea1-85a7665ceb5b.png">
